### DISCUSSION TITLE
In-Network file Service_Code (Place of Service Codes) for Facility services - (DRG & Revenue code)

### DISCUSSION THREAD

**Discussion by vijayanandramamoorthy** [CMSgov/price-transparency-guide#105] [Q&A] [answered]
Since Serivce_Code (Place of Service Code) is applicable only for professional services, what do we report for the Service_Code field, for the Facility Services? (for eg. DRG & Revenue Codes pricing)
DRG & Revenue codes are typically configured to pay based on type of bill since they are billed on a facility claim. Would a type of bill code be more appropriate rather than the place of service, for these codes?

thanks

**Comment by angelamarie0975** [comment]
@shaselton 
Can we please get an answer about Service_Code in general? Is Service_Code only required for Professional services? Can we get some clarification on Service_Code please? There are Place of Service Codes 0-99 with CMS - for facility service examples, there are: 21 - Inpatient Hospital, 22 - Outpatient Hospital, 23 -  Hospital, 24 - Ambulatory Surgical Facility. For our company, our negotiated rates can be different for Facility vs Non-Facility and we are able to determine that based off of our contracting rules for us to display at that level. 

However, as there are so many service codes with CMS and potentially, they could all have the same negotiated rate, there was a discussion about the possibility of adding Service_Code as an array so we didn't have so many duplicated rates. There was also talk of possibly just having this at Facility vs Non-Facility. Is there any update to that? 

Also - I see that in the requirements for Service_Code, it does state 'professional claim', but it says it's required, so are we not needing to put the service code on the Institutional Provider Contract data? 
From the schema for both In-Network and Allowed Amounts - service_code | Place of Service Code | String | The CMS-maintained two-digit code that is placed on a professional claim to indicate the setting in which a service was provided | Yes

I will post this same note to that discussion as well.

**Reply by vishalpatel2** [reply]
@tdfow  we hope you can help us out here as I see you are the active contributor. The main question boil downs to is what should be the place of service code for facilities/hospital rates given description is calling out that it is only for professional claims?

**Reply by tdfow** [reply]
I am not associated with CMS, so I can only provide comments regarding this-- For facility related rates, possibly associate a translation to a Place of Service value based on the Type of Bill related data.  

The schema has the Place of Service value as required data---so the value in the file seems to be related to the Place of Service value--but for facility type rates, it seems you would have to associate what Place of Service values to use.  For example, skilled nursing TOB rates would then associate to Place of Service value 31

**Reply by ryan-thomas-peterson** [reply]
@tdfow , this leaves an opening for significant implementation variation.  The Place Of Service only appears on professional claims.  Mapping from Type of Bill without some standardized resource means we're each left doing our own thing.  The current schema appears focused on professional rates.  The requirement of a level 1 NPI and POS code suggest that facility rates either weren't actively considered or are now trying to be 'fit' into this schema or are intended to be 'baked' into the professional rate.   It's clear from the rule that facility fees should be included, I just was hopeful for some more guidance on designating these rates in conjunction with required fields that are not in contracts or on claims.

**Reply by tdfow** [reply]
@ryan-thomas-peterson, agree.   Without guidance, that is the only suggestion I had since it is a required field (based on the current schema) and that we know there are rates needed beside the professional related ones.  We know that the first byte of the facility code identifies the type of facility---so that at least gives a value that can be mapped to a POS value that can be used.  Agree if there is only a required POS, then there should be guidance on what values should be used for facility claims based on 'x' or provide another option for facility rates---or make service code optional

**Comment by rgouldkc** [comment]
NUBC defines the Type Of Bill values. We are ignoring the leading 0 digit of the 4 digit code which not submitted in an 837. The next 2 digits define place of service such as Hospital Inpatient (011x), Hospital Inpatient Medicare Part B only (012x), Hospital Outpatient (013x). The last digit  is a bill frequency code. The UB04 Specifications manual which must be licensed from NUBC has all the details. AT least for us this is what we will use. The documentation also indicates which are Inpatient vs Outpatient.

**Reply by ryan-thomas-peterson** [reply]
If you take those two digits in the example of a Hospital Inpatient (11) that certainly conflicts with the definition of the schema "The CMS-maintained two digit code that is placed on a professional claim to indicate the setting in which a service was provided".  11 for professional claims is an office setting.  One can 'guess' based on the TOB that IP claims really ought to be a Place of Service of Inpatient Hospital but then we potentially confuse professional IP services with facility contracted rates. 

I'd propose something like the following: IP and OP for inpatient and outpatient facility settings, or a Type Of Bill array (but only carrying the first two digits for Type of Facility and Type of Care)) or a switch indicating if the POS array is intended to be interpreted as Types Of Bill or Places of Service.

**Comment by shaselton-usds** [comment, marked as answer]
[`service_code`](https://github.com/CMSgov/price-transparency-guide/tree/master/schemas/in-network-rates#negotiated-price-object) is only required for the "professional" billing class.
